doc/source/whatsnew/v0.18.0.txt

```diff
@@ -33,6 +33,8 @@ Other enhancements
 
 - Handle truncated floats in SAS xport files (:issue:`11713`)
 - ``read_excel`` now supports s3 urls of the format ``s3://bucketname/filename`` (:issue:`11447`)
+- A simple version of Panel.round() is now implemented (:issue:`11763`)
+- For Python 3.x, round(DataFrame), round(Series), round(Panel) will work (:issue:`11763`)
```

Comment:
in API breaking show that we removed the ``out`` arg from ``Series.round`` (same issue number)

use double backticks no things like ``round(DataFrame)`` and such